FC-0068 docs: Update How-tos edunext 1 (#643)
openedx/docs.openedx.org: source/educators/how-tos/advanced_features/configure_content_experiments.rst

.. _Configure Your Course for Content Experiments:

#####################################################
Configure Your Course for Content Experiments
#####################################################
#############################################

.. tags:: educator, how-to

This section provides instructions for configuring your course to use
:ref:`content experiments<Overview of Content Experiments>`.
This how-to provides instructions for configuring your course to enable
:ref:`content experiments<Overview of Content Experiments>`. Let's start.

.. contents::
  :local:
  :depth: 1

.. _Enable Content Experiments:

****************************************
Enable Content Experiments
****************************************

To enable content experiments in your course, you add ``split_test`` to the
**Advanced Module List** in Advanced Settings.

> .. note::

Reviewer: Why delete this note, and the L22? It might not be obvious how the phrase split_test relates to experimnets, and L22 tells in advance what the steps will walk you through

Author: This information is passed to the specific How-to create_content_experiment.rst (source/educators/how-tos/advanced_features/create_content_experiment.rst). I had to break this how-to into several documents, but I can add that information here again if necessary.

Reviewer: I would add these back here in this place. I don't think it's good to expect that someone reads through every related how-to article in order to fully understand a component.

Reviewer: @Apgomeznext - once this comment is addressed we can merge this PR.

Author: Comment addressed.

.. _Configure Your Course for Content Experiments:

Configure Your Course for Content Experiments
#############################################

.. tags:: educator, how-to

This how-to provides instructions for configuring your course to enable
:ref:`content experiments<Overview of Content Experiments>`. 

> To enable content experiments in your course, you must ``split_test`` to the

Reviewer: ```suggestion
To enable content experiments in your course, you must add ``split_test`` to the
```

#. In the **Advanced Module List** field, add ``"split_test"``. Be sure that
   you include the double quotation marks.

>    If you have multiple values, ensure that commas separate them

Reviewer: ```suggestion
   If you have multiple values, ensure that they are separated by commas
```

> .. _Set up Group Configurations in edX Studio:

Reviewer: ```suggestion
.. _Set up Group Configurations in Studio:
```
or "Open edX Studio", I don't mind either way
